mmv1/products/securitycentermanagement/FolderSecurityHealthAnalyticsCustomModule.yaml

```diff
@@ -0,0 +1,232 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'FolderSecurityHealthAnalyticsCustomModule'
```

Comment:
The name FolderSecurityHealthAnalyticsCustomModule results in the new resource having this name:
- google_scc_management_folder_security_health_analytics_custom_module

But in your acceptance tests you are referring to the resource as:
- google_scc_management_organization_folder_security_health_analytics_custom_module

Could you please either update the name here (and the file name) _or_ update the acceptance test to use the correct resource name?

Reply:
Thank you for the insight. Addressed.

Reply:
Thanks!